src/app/playground/components/ArrowLinks.tsx

```diff
@@ -0,0 +1,71 @@
+import { FingerprintJSPro } from '@fingerprintjs/fingerprintjs-pro-react';
+import { EventResponse } from '@fingerprintjs/fingerprintjs-pro-server-api';
+import Link from 'next/link';
+import { FunctionComponent, useRef } from 'react';
+import { ExternalLinkArrowSvg } from '../../../client/img/externalLinkArrowSvg';
+import styles from '../playground.module.scss';
+import { TEST_IDS } from '../../../client/testIDs';
+
+export const DocsLink: FunctionComponent<{ children: string; href: string; style?: React.CSSProperties }> = ({
+  children,
+  href,
+  style,
+}) => {
+  // Prevent the arrow from being the only element on a new line
+  const lastWord = children.split(' ').pop();
+  const leadingWords = children.split(' ').slice(0, -1).join(' ');
+  return (
+    <Link href={href} target='_blank' className={styles.docsLink} style={style}>
+      {leadingWords}{' '}
+      <span style={{ whiteSpace: 'nowrap' }}>
+        {lastWord}
+        <ExternalLinkArrowSvg className={styles.externalLinkArrow} />
+      </span>
+    </Link>
+  );
+};
+
+type PropertyName = keyof EventResponse['products'] | keyof FingerprintJSPro.ExtendedGetResult;
+
+export const JsonLink: FunctionComponent<{
+  children: string;
+  propertyName: PropertyName;
+  elementOrder?: 'first' | 'last';
+}> = ({ children, propertyName, elementOrder }) => {
+  const timeout = useRef<NodeJS.Timeout | undefined>();
+
+  // Prevent the arrow from being the only element on a new line
+  const lastWord = children.split(' ').pop();
+  const leadingWords = children.split(' ').slice(0, -1).join(' ');
```

Comment:
We perform split twice here, can we assign the split value to separate variable and re-use it here?

Reply:
Good catch, thanks!  https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/pull/153/commits/67224e3f10a69c09eb4ac539d9530dedbaa4a1a4

Reply:
Yep, looks good, thanks!